code/game/objects/items/weapons/twohanded.dm

```diff
@@ -411,3 +411,28 @@
 	if(src == user.get_active_hand()) //update inhands
 		user.update_inv_l_hand()
 		user.update_inv_r_hand()
+
+
+//GREY TIDE
+/obj/item/weapon/twohanded/spear/grey_tide
+	icon_state = "spearglass0"
+	name = "\improper Grey Tide"
+	desc = "Recovered from the aftermath of a revolt aboard Defense Outpost Theta Aegis, in which a seemingly endless tide of Assistants caused heavy casualities among Nanotrasen military forces."
+	force_unwielded = 15
+	force_wielded = 25
+	throwforce = 20
+	throw_speed = 4
+	attack_verb = list("gored")
+	var/list/spear_owners = list()
+/obj/item/weapon/twohanded/spear/grey_tide/afterattack(atom/movable/AM, mob/living/user, proximity)
+	..()
+	if(!(user in spear_owners))
+		user.faction += "greytide([rand(1, 1000)])"
+		spear_owners += user
```

Comment:
You don't need this `spear_owners` dance if you just use `user.faction |= "greytide\ref[user]"` or something of the sort.